sass/theme.scss

```diff
@@ -115,7 +119,7 @@ select {
 }
 
 .add {
-  text-align: center;
+  text-align: left;
```

Comment:
Just remove the text alignment for `.sidebar` we don't center anything anyway. After that you can remove all the alignments in sidebar children elements.